content/en/docs/how-tos/add-jobs-to-testgrid.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Testgrid"
+date: 2020-10-28T11:14:39-04:00
+draft: false
+---
+
+We talk about two distinct components in this doc - testgrids and release-informing/blocking/broken jobs. 
```

Comment:
I would just remove this passage. Testgrid might qualify as a "component", release-informing/blocking/broken jobs definiutely due not.